tests/graph/test_basic.py

```diff
@@ -365,8 +365,17 @@ def test_eval_with_strings_no_match(self):
     def test_eval_kwargs(self):
         with pytest.raises(UnusedInputError):
             self.w.eval({self.z: 3, self.x: 2.5})
+        with pytest.warns(
+            UserWarning,
+            match="pytensor.function was asked to create a function",
+        ):
+            self.w.eval({self.z: 3, self.x: 2.5}, on_unused_input="warn")
         assert self.w.eval({self.z: 3, self.x: 2.5}, on_unused_input="ignore") == 6.0
 
+        # regression test for https://github.com/pymc-devs/pytensor/issues/1084
+        q = self.x + 1
+        assert q.eval({"x": 1, "y": 2}, on_unused_input="ignore") == 2.0
```

Comment:
Hmm so this is when y can be found but isn't needed? I thought ot was for when y can't be found. Makes more sense

Reply:
Right, it's when `y` is supplied but not needed.